src/plugins/intel_gna/CMakeLists.txt

```diff
@@ -6,8 +6,13 @@ if (NOT ENABLE_INTEL_GNA)
     return()
 endif()
 
+
 set(TARGET_NAME "openvino_intel_gna_plugin")
 
+if (ENABLE_INTEL_GNA_DEBUG AND (CMAKE_BUILD_TYPE STREQUAL "Debug"))
+    add_compile_definitions(GNA_DEBUG)
+endif()
```

Comment:
single Build type (make/Linux) vs multiple Bulid config (VS/Windows) behavior problem here.
see https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html

Reply:
I think we should enable setting this option for all build types.